target/linux/siflower/image/Makefile

```diff
@@ -0,0 +1,36 @@
+# SPDX-License-Identifier: GPL-2.0-only
+#
+# Copyright (C) 2012-2015 OpenWrt.org
+# Copyright (C) 2016-2017 LEDE project
+
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/image.mk
+
+# default all platform image(fit) build
+define Device/Default
+  PROFILES = Default $$(DEVICE_NAME)
+  KERNEL = kernel-bin | lzma | \
+	fit lzma $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb
+  KERNEL_INITRAMFS = kernel-bin | lzma | \
+	fit lzma $$(KDIR)/image-$$(firstword $$(DEVICE_DTS)).dtb with-initrd
+  FILESYSTEMS := squashfs
+  DEVICE_DTS_DIR := $(DTS_DIR)
+  IMAGES := sysupgrade.bin
+  IMAGE/sysupgrade.bin := append-kernel | append-rootfs | pad-rootfs | \
+	append-metadata
+endef
+
+define Device/UbiFit
+  KERNEL_IN_UBI := 1
+  IMAGES := nand-factory.ubi nand-sysupgrade.bin
+  IMAGE/nand-factory.ubi := append-ubi
+  IMAGE/nand-sysupgrade.bin := sysupgrade-tar | append-metadata
+endef
```

Comment:
This isn't used by any boards and can be dropped. Also, the fitblk by @dangowrt is better than sysupgrade-tar. You can take a look at the mediatek target for examples.